src/systemtask/SystemTask.cpp

```diff
@@ -228,6 +233,12 @@ void SystemTask::Work() {
           motorController.SetDuration(35);
           displayApp->PushMessage(Pinetime::Applications::Display::Messages::NewNotification);
           break;
+        case Messages::OnTimerDone:
+          if (isSleeping && !isWakingUp)
```

Comment:
Missing braces. This is a clang-tidy check you can enable. 

(If you use CLion, you can also enable this integration in the settings)